src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/react-native-installation.mdx

```diff
@@ -0,0 +1,312 @@
+      </td>
+    </tr>
+
+    <tr>
+      <td id="attributes-events-insights">
+        Report custom attributes and events
+      </td>
+
+      <td>
+        There are several ways to add custom attributes and events. For more about which would be the best method to use and why, see [Add custom data](/docs/mobile-monitoring/new-relic-mobile/maintenance/add-custom-data-new-relic-mobile).
+
+        * Create an attribute: [`setAttribute`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/setattribute-android-agent-api)
+        * Increment an attribute count: [`incrementAttribute`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/incrementattribute-android-agent-api)
+        * Remove an attribute: [`removeAttribute`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/removeattribute-android-agent-api)
+        * Remove all attributes: [`removeAllAttributes`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/removeallattributes-android-agent-api)
+        * Record custom event: [`recordCustomEvent`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/recordevent-android-agent-api)
+        * Record a breadcrumb event: [`recordBreadcrumb`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/recordbreadcrumb)
+        * Set the maximum size of an event pool: [`setMaxEventPoolSize`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/setmaxeventpoolsize-android-agent-api)
+        * Set maximum time agent stores events in memory: [`setMaxEventBufferTime`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/setmaxeventbuffertime-android-agent-api)
+        * Return the current session's ID: [`currentSessionId`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/currentsessionid-android-agent-api)
+        * Set custom user ID for associating sessions with events and attributes: [`setUserId`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/setuserid-android-agent-api)
+      </td>
+    </tr>
+
+    <tr>
+      <td id="track-custom">
+        Track custom network requests and failures
+      </td>
+
+      <td>
+        Use these methods:
+
+        * Record HTTP transactions at varying levels of detail: See [`noticeHttpTransaction`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/noticehttptransaction-android-agent-api).
+        * Record network failures: See [`noticeNetworkFailure`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/noticenetworkfailure-android-agent-api).
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Record a handled exception as an event, including context
+      </td>
+
+      <td>
+        See `recordHandledException()`.
+      </td>
+    </tr>
+  </tbody>
+</table>
+  </Collapser>
+</CollapserGroup>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Java Script Errors"
```

Comment:
should be `JavaScript`. I'd also just do 'errors' lower case (unless i'm missing something about why it should be capitalized)